data/human/human missions.txt

```diff
@@ -7453,3 +7453,442 @@ mission "Adelita 5"
+			choice
+				`	"That makes sense. Let's discuss the deal more after Oblate."`
+				`	"Thanks for the overview, but I'm still not interested. I'm out."`
+					decline
+			label accepting
+			`	Howard nods with approval. "Excellent. The Marauder ship was last seen near Oblate, in the outer rim of that system. It's likely to be well armed, so stay sharp. Once you've disabled it, head back here and I'll make sure you're properly compensated."`
+			branch blewupgolden
+				has "syndicate business golden jewel dead"
+			`	He cracks a smile. "I remember the first time we met you left that Vanguard alive. Same deal here, but disable the Marauder. Hopefully that Vanguard's not still around.`
+				goto ending
+			label blewupgolden
+			`	He leans in slightly, lowering his voice. "Just be careful. We don't want to destroy the ship. Cripple it so we can study it, but don't blow it into oblivion like you did to that Vanguard when we first met.`
+			label ending
+			`	"Well, that's all I have for you. Goodbye and good luck!"`
+				accept
+
+	on enter "Moktar"
+		"reputation: Independent (Killable)" = -1000
+	npc
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"
+		to despawn
+			has "syndicate business golden jewel dead"
+		on encounter
+			conversation
+				`When you enter Moktar, your sensors detect the same pirate Vanguard from when you first rescued Howard. Your ship is immediately hailed, and the pirate opens communication.`
+				`	"I should have destroyed you when I had the chance. Prepare to meet your end!"`
+				choice
+					`	"Maybe I should have destroyed you!"`
+					`	"Uhhh... who are you again?"`
+						goto whomst
+				`	"Argh, you slithering, spineless snake! I'll tear your ship apart piece by piece!"`
+					goto power
+				label whomst
+				`	"Bah, you banal, bumbling baboon! Prepare to meet your doom!"`
+				label power
+				`	You can't help feeling a sense of deja vu as the Vanguard's engines roar to life and you see over half a dozen fixed guns take aim at your ship, ready to strike.`
+		on kill
+			set "syndicate business golden jewel dead"
+			dialog "The crew of the Golden Jewel are no more. You think you can make out the pirate captain drifting in space nearby."
+
+	on enter "Oblate"
+		"reputation: Independent (Killable)" = -1000
+	npc disable save
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Oblate"
+		ship "Marauder Quicksilver (Engines) (Safe)" "Pilgrim Shadow"
+		on disable
+			conversation
+				`The ship has been disabled. You receive a message from Howard almost immediately:`
+				`	"<first> <last>, you've done it again! Meet back at <destination> to discuss."`
```

Comment:
I think "discuss" here doesn't work as well grammatically as a different word that means the same thing? Maybe the way the sentence is put?
```suggestion
				`	"<first> <last>, you've done it again! Meet me on <destination> to talk about this victory."`
```
This sentence somehow feels more natural to me than the way it is now.

Reply:
I was going for the corporate feel that uses phrases like "lets discuss," "set up a meeting to discuss," etc, but I'm good to change it here if that feels better.

Reply:
Well, I don't know. I think it sounds better my way, but I don't know how "big business" is really supposed to operate. If discuss is used for corporations, then by all means leave it in that way.

Reply:
Well there's a million usable corporate phrases, so it'd still be better to pick one that feels natural/better. Put together another few options with corporate jargon, also throwing in a few sports business ones that could fit better with a salesman vs generic business. Do you like any of these?
`	"<first> <last>, you've done it again! Let's circle back on <destination> to align on next steps."`

`	"<first> <last>, this is a game changer! Let's sync up on <destination> to debrief."`

`	"<first> <last>, another home run! Meet me on <destination> to touch base."`